src/modules/45-dashboards/pages/home/HomePage.tsx

```diff
@@ -337,10 +335,27 @@ const RenderDashboardTags: Renderer<CellProps<DashboardInterface>> = ({ row }) =
   return TagsRenderer(data)
 }
 
+const getBreadcrumbLinks = (folderDetail: { resource: string }, accountId: string, folderString: string) => {
```

Comment:
You should add the return type
```suggestion
const getBreadcrumbLinks = (folderDetail: { resource: string }, accountId: string, folderString: string): Breadcrumb[] => {
```